### PR TITLE
{toolchain} [GNU] Added iompi-2015.03 and iomkl-2015.03 (REVIEW)

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ade0da37ca0ae74e3c55 for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
@wpoely86: please add an HPL easyconfig using this toolchain by means of test case?

File: easybuild/easyconfigs/h/hwloc/hwloc-1.11.1-iccifort-2015.3.187-GNU-4.9.3-2.25.eb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ NUMA memory nodes, sockets, shared caches, cores and simultaneous multithreading. It also gathers various
+ system attributes such as cache and memory information as well as the locality of I/O devices such as
+ network interfaces, InfiniBand HCAs or GPUs. It primarily aims at helping applications with gathering
+ information about modern computing hardware so as to exploit it accordingly and efficiently."""
+
+toolchain = {'name': 'iccifort', 'version': '2015.3.187-GNU-4.9.3-2.25'}
+
+source_urls = ['http://www.open-mpi.org/software/hwloc/v%(version_major_minor)s/downloads/']
+sources = [SOURCE_TAR_GZ]
+
```
Comment (boegel) [inline easybuild/easyconfigs/h/hwloc/hwloc-1.11.1-iccifort-2015.3.187-GNU-4.9.3-2.25.eb]: `numactl` is missing a dep, see other recent hwloc easyconfigs?

File: easybuild/easyconfigs/i/imkl/imkl-11.2.3.187-iompi-2015.03.eb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+for science, engineering, and financial applications that require maximum performance. Core math functions include
+BLAS, LAPACK, ScaLAPACK, Sparse Solvers, Fast Fourier Transforms, Vector Math, and more."""
+
+toolchain = {'name': 'iompi', 'version': '2015.03'}
+
+sources = ['l_mkl_%(version)s.tgz']
+
+dontcreateinstalldir = 'True'
+
+interfaces = True
```
Comment (boegel) [inline easybuild/easyconfigs/i/imkl/imkl-11.2.3.187-iompi-2015.03.eb]: this is the default, no need to add it?

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/954bfa232f3be45e8579 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 0 out of 0 (7 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ee27fa11256376a50cf8 for a full test report.

**Comment by wpoely86** [comment]
@boegel something went wrong in the test?

**Comment by boegel** [comment]
@wpoely86: yeah, no `--force`, retrying :)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/fa51f698b9a4a738d696 for a full test report.

**Comment by boegel** [comment]
going in, thanks @wpoely86!

**Pull request merged**